packages/ui/src/Buttons/SubmitButton/__test__/__snapshots__/SubmitButton.snapshot.test.tsx.snap

```diff
@@ -2,9 +2,13 @@
 
 exports[`Component matches snapshot 1`] = `
 <button
-  className="button-submit"
+  className="dz-button  primary filled medium"
```

Comment:
Could you please check why we have two spaces here between dz-button and primary? It might be coming from the Button component.